homeassistant/components/meteoalarm/binary_sensor.py

```diff
@@ -81,7 +82,9 @@ def device_state_attributes(self):
     @property
     def icon(self):
```

Comment:
Remove `icon()`. It's a binary sensor and the visual representation should be handed by the device class.

Reply:
May I suggest that you discuss your refactor suggestion with the component owner and that a separate/different PR be created.
I'm not in a position to argument on the change you propose, and they definitely would change the purpose of the PR.